docs/Gestor_Dependencias.md

```diff
@@ -0,0 +1,16 @@
+# Elección del gestor de dependencias.
+
+Teniendo en cuenta [el issue](https://github.com/MauronMP/PMP_IV/issues/17) y la [HU0](https://github.com/MauronMP/PMP_IV/issues/7) #7, se quiere elegir un gestor de dependencias como parte del objetivo 3.
+
+Hay distintas opciones para el gestor de dependencias:
+- **venv.**  Es una parte de la librería estándar de python, en caso de tener una versión superior a la 3.3, ya se tiene instalado. Usa un entorno virtual que incluye su propia instalación de python.
```

Comment:
Dediqué unos 5 minutos en clase explícitamente a explicar qué es lo que determinaba la elección del gestor de dependencias en Python. Os agradecería que prestárais algo de atención, o si no, que busquéis cuales son las mejores prácticas en gestión de dependencias en Python este año y por qué es así.

Reply:
Has cambiado esto de sitio para que desaparezca el comentario, pero siguen diciendo lo mismo.
La mejor forma de optimizar tu tiempo y el mío sería que hicieras caso a las objeciones que se te hacen y actuaras con respecto a ellas.

Reply:
Buenas, quizás no lo estoy entendiendo bien, he estado buscando distintos gestores de dependencias, había puesto 4, pero he visto mejor poner 3, no quería desaparecer el comentario. De los 3 que he visto, he comparado los tres y he sacado la conclusión de cuál es mejor y conviene para el proyecto. ¿Qué faltaría entonces?

Reply:
Que no se trata de poner 4 o 3 o 355. Se trata de seguir las buenas prácticas en Python, que me tiré 5 minutos explicando en clase. Que está claro que no logré transmitir nada, pero ya que te lo esoty diciendo, podías al menos buscar un poco de información sobre el tema. Que tampoco entiendo como "que busquéis cuales son las mejores prácticas en gestión de dependencias en Python este año y por qué es así" se interpreta con "tengo que poner 3 gestores de dependencias en vez de 4", tampoco, pero ese es otro tema.
Y por otro lado, es que ni siquiera son gestores de dependencias, sino de entornos virtuales (salvo pipenv, pero no estoy seguro)

Reply:
Vale, lo miro entonces y lo rectifico.